### PR TITLE
count all lines && fixed ducc scores

### PR DESCRIPTION
**Pull request by aboft**
fix to count all the lines and not just count regular users messages. Upon fixing this I found out the scores feature stopped working due to the parsing. Corrected logic in the scores function.

### CODE REVIEW

File: backend/database/DuccScores.ts [hunk 1]
```diff
@@ -5,8 +5,10 @@ class DatabaseDuccUtils {
     let tableName: string;
     duccType == 'friend' ? (tableName = 'friend_scores') : (tableName = 'killer_scores');
     scores.forEach(async (scoreString) => {
+      scoreString = scoreString.replace(/[\x02|\u200B]/g, '').trim();
       const score: string[] = scoreString.split(':');
-      const duccs: number = parseInt(score[1].replace(',', ''));
+      const duccs: number = parseInt(score[1].trim());
+      //score[0] = score[0].replace(/^[a-zA-Z0-9]/, '');
```
Comment (LoLei) [inline backend/database/DuccScores.ts:11]: commented line

File: backend/irc/IrcMessageProcessor.ts [hunk 2]
```diff
@@ -211,16 +211,20 @@ class IrcMessageProcessor {
       myEmitter.emit('line');
       // Process bot messages
       // Process ducc stats
+      console.log('PRIVMSG PARSE', nick, msg);
       if (msg.match(/Duck \w{6} scores in #/i) && nick === 'gonzobot') {
+        console.log('inside the ducc parse');
```
Comment (LoLei) [inline backend/irc/IrcMessageProcessor.ts]: These logs could be removed

File: backend/database/DuccScores.ts (hunk 1 above)
Comment (LoLei) [inline backend/database/DuccScores.ts]: commented line

**Review by LoLei — CHANGES_REQUESTED** [review]

File: backend/irc/IrcMessageProcessor.ts (hunk 2 above)
Comment (aboft) [reply backend/irc/IrcMessageProcessor.ts]: these logs are removed

File: backend/database/DuccScores.ts (hunk 1 above)
Comment (aboft) [reply backend/database/DuccScores.ts]: no commented line

File: backend/database/Messages.ts [hunk 3]
```diff
@@ -88,7 +88,10 @@ class DatabaseMessageUtils {
       if (lineCountExists.length < 1) {
         await knex('line_counts').insert({ count: 0, botLines: 1, date: this.formatDate(new Date()) });
```
Comment (LoLei) [inline backend/database/Messages.ts:89]: Shouldn't the `count` be set to 1 here as well? As it's done below as well.

Comment (aboft) [reply backend/database/Messages.ts:89]: made an emergency commit after merging. didn't catch this in time for PR but master has this updated correctly.

**Pull request merged**